runelite-client/src/main/java/net/runelite/client/plugins/driftnet/DriftNetPlugin.java

```diff
@@ -204,6 +226,19 @@ public void onGameTick(GameTick tick)
 			return;
 		}
 
+		for (DriftNet net : NETS)
+		{
+			// untag fish which are next to a net that is not accepting fish right now
+			// or was not accepting fish previous tick (When a fish shoal is 2 tiles adjacent
+			// to a drift net and is moving to a net that is just being setup it will be denied
+			// even though the net is currently in the CATCHING status)
+			if (net.isNotAcceptingFish() || net.wasNotAcceptingFishPrevTick())
+			{
+				taggedFish.entrySet().removeIf(entry -> checkFishOnTiles(entry.getKey(), net.getAdjacentTiles()));
```

Comment:
i was thinking that this could go in a method too so you could use the `removeIf` below in a `removeIf(expired || closeToUnsetNet)` (naming idk)